package.json

```diff
@@ -9,7 +9,9 @@
     "axios": "^1.3.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.6",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
add a title to your index.html file! and add a favicon! https://formito.com/tools/favicon makes that super easy